.github/workflows/tests.yml

```diff
@@ -89,6 +89,13 @@ jobs:
           working-directory: aqua-tests
           flags: "--save-dev"
 
+      - name: Set aqua version in fluence.yml
+        if: inputs.aqua-version != 'null'
+        env:
+          NPM_CONFIG_REGISTRY: "https://npm.fluence.dev"
+        run: |
+          sed -i 's/"@fluencelabs\/aqua": .*/"@fluencelabs\/aqua": ${{ inputs.aqua-version }}/g' fluence.yaml
```

Comment:
Can you check afterwards that version was actually used in `.fluence`?

Reply:
also, there should be a `fluence dependency install` step, I don't see it, maybe I'm blind

Reply:
without `fluence dependency install`, I'm not sure how it will work. Maybe `fluence run` will install it, but that feels awkward

Reply:
It should run dependency install when fluence-cli first runs. But I guess it would be better to add it as a step.

Regarding checking if it was actually used I propose to do it in another task by creating action `set-fluence-cli-dep-version` or something that will abstract all the setting and checking. Here it is just a quick fix so registry e2e actually use aqua snapshots built during e2e.

Reply:
💯 agree. Create a linear task on set dep version. 🙇